cumulus/mytaskflows.py

```diff
@@ -93,7 +92,7 @@ class ChordTaskFlow(taskflow.TaskFlow):
     This taskflow has a "fanout" an also a chord, at task4
     """
     def start(self):
-        task1.delay(self)
+        task4.delay(self)
 
```

Comment:
This should be task1,  i just switched to task4 to jump right the chord for debugging.  FIxed in latest commit